### DISCUSSION TITLE
Unhandled Runtime ErrorError: React.Children.only expected to receive a single React element child.

### DISCUSSION THREAD

**Discussion by catherineangela** [vercel/turborepo#2510] [Help] [answered]
Call Stack
Object.onlyChild [as only]
/_next/static/chunks/node_modules__next@13.0_7e5f88.0@next.js (1416:23)
LinkComponent
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (207:45)
renderWithHooks
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (28178:28)
updateForwardRef
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (30549:32)
mountLazyComponent
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (31145:33)
beginWork
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (32202:32)
beginWork$1
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (36901:28)
performUnitOfWork
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (36227:24)
workLoopSync
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (36152:17)
renderRootSync
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (36121:21)
recoverFromConcurrentError
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (35722:30)
performConcurrentWorkOnRoot
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (35668:38)
workLoop
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (39569:41)
flushWork
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (39548:40)
MessagePort.performWorkUntilDeadline
/_next/static/chunks/node_modules__next@13.0.0@next_link.js (39375:45)

**Comment by nathanhammond** [comment, marked as answer]
This is not a Turbo error, it's an authoring error.
